snappi_ixnetwork/ixnetworkapi.py

```diff
@@ -136,16 +142,22 @@ def _request_detail(self):
     def set_config(self, config):
         """Set or update the configuration
         """
+        if isinstance(config, (type(self.config()),
```

Comment:
`self.config()` is costly, so save it in a var so it could be used during deserialization.

Reply:
Added as variable